book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rental = []
+  end
+
+  def add_rental(date, _rental)
+    Rental.new(date, self, person)
```

Comment:
- [ ] Kindly make sure to pass the right parameters to the `add_rental` method. Currently, you're not passing the `person` parameter to the `add_rental` method but you're passing it to the new rental. Kindly fix this.
- [ ] Please be mindful of the order of parameters passed in the new rental. The rental's constructor has the following paramters passed in it:
```
def initialize(date, person, book)
```
- Please follow the same order of parameters passed.